src/test/groovy/edu/oregonstate/mist/locations/frontend/LocationDAOTest.groovy

```diff
@@ -0,0 +1,84 @@
+package edu.oregonstate.mist.locations.frontend
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import edu.oregonstate.mist.locations.frontend.db.LocationDAO
+import io.dropwizard.testing.junit.DropwizardAppRule
+import org.junit.BeforeClass
+import org.junit.ClassRule
+import org.junit.Test
+
+class LocationDAOTest {
+    private static LocationDAO locationDAO
+
+    private ObjectMapper mapper = new ObjectMapper()
+
+    private final String SINGLE_RESOURCE_ID = "831ed9ce6311601afba57934adea7a8e"
+
+    @ClassRule
+    public static final DropwizardAppRule<LocationsFrontendConfiguration> APPLICATION =
+            new DropwizardAppRule<LocationsFrontendConfiguration>(
+                    LocationsFrontEndApplication.class,
+                    new File("configuration.yaml").absolutePath)
```

Comment:
did this line solve the issue with `configuration.yaml` in `src/main/resources`?